core/src/main/java/org/infinispan/configuration/cache/SingletonStoreConfigurationBuilder.java

```diff
@@ -11,6 +11,11 @@
 import org.infinispan.commons.configuration.attributes.AttributeSet;
 import org.infinispan.configuration.global.GlobalConfiguration;
 
+/**
+ * @deprecated Singleton writers will be removed in 9.1. If it is desirable that all nodes don't write to the underlying store
```

Comment:
If we deprecate them in 9.0, we probably won't be able to remove them until 10... Am I right @tristantarrant ?